.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.5', '2.6', '2.7', '3.0', '3.1']
```

Comment:
Can we stop supporting Ruby 2.5 and 2.6 because they are also EOL?

Reply:
In my opinion, we don't have to stop supporting 2.5 and 2.6 until we use features they don't support.